### ISSUE TITLE
4.0 `findConfiguration()` replacement

### ISSUE THREAD

**Issue by unional** [palantir/tslint#1765]
What is the replacing workflow of `findConfiguration()`?

I'm using it to load different config in my test.
https://github.com/unional/tslint-config-unional/blob/master/gulpfile.js#L13

Thanks. 🌷

**Comment by nchen63** [comment]
does `tslint.Configuration.findConfiguration()` work?

**Comment by unional** [comment]
Seems like it is not working correctly somehow.

However I do have my problem solved. It turns out now `gulp-tslint` support passing in path directly to it:

```js
  let config = 'standard'
  return gulp.src(`spec/${config}/*.pass.ts`)
    .pipe(gulpTslint({
      configuration: `./${config}.js`,
      formatter: 'verbose'
    }))
    .pipe(gulpTslint.report())
```

When I do this, it does not work:
```js
  let config = 'standard'
  return gulp.src(`spec/${config}/*.pass.ts`)
    .pipe(gulpTslint({
      configuration: tslint.Configuration.findConfiguration(`./${config}.js`),
      formatter: 'verbose'
    }))
    .pipe(gulpTslint.report())
```

**Comment by nchen63** [comment]
cool, will close the issue then
